bin/check-ec2-filter.rb

```diff
@@ -0,0 +1,144 @@
+#! /usr/bin/env ruby
+#
+# check-ec2-filter
+#
+# DESCRIPTION:
+#   This plugin retrieves EC2 instances matching a given filter and
+#   returns the number matched
```

Comment:
This description should better explain that the check alerts on counts of instances returned.  This description sounds more like the metrics plugin